app/src/main/java/com/marvelapp/presentation/viewmodel/MainMenuViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.marvelapp.presentation.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+@HiltViewModel
+class MainMenuViewModel @Inject constructor() : ViewModel() {
```

Comment:
as this constructor is empty, this is not necessary

Reply:
Solved